src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/GyroIOs/Gyro.java

```diff
@@ -0,0 +1,53 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO.GyroIOs;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+
+public class Gyro {
+
+    private GyroIO io;
+    private final GyroIOInputsAutoLogged inputs = new GyroIOInputsAutoLogged();
+
+    public Gyro() {
+        switch (RobotMap.ROBOT_TYPE) {
+            case Frankenstein:
+            case pegaSwerve:
```

Comment:
Also, if we log when using the simulation gyro, why not when using a "real" gyro